src/AssetManager/View/Helper/Asset.php

```diff
@@ -92,16 +109,24 @@ public function __invoke($filename)
             $cacheConfig = $this->config['caching']['default'];
         }
 
-        // if nothing done, return the original filename
-        if (!isset($cacheConfig['options']['dir'])) {
-            return $filename;
-        }
-
         // query string params
         $queryString = isset($this->config['view_helper']['query_string'])
             ? $this->config['view_helper']['query_string']
             : '_';
```

Comment:
I don't think we should care. It's a logical default. If you wish to destroy it on purpose it's your own fault.